lib/internal/Magento/Framework/App/Response/Http/FileFactory.php

```diff
@@ -102,4 +107,19 @@ public function create(
         }
         return $this->_response;
     }
+
+    /**
+     * Returns file content for writing.
+     *
+     * @param string|array $content
+     * @return string
+     */
+    private function getFileContent($content)
+    {
+        if (isset($content['type']) && $content['type'] == 'string') {
```

Comment:
We should check if `$content` is an array before checking the `isset`. Not sure that PHP will not complain about this.
In this case is not strictly required, but I suggest always to use the `===` strict comparison.

Reply:
`isset` wont generate a notice in that case if the index does not exist. Also, it's applicable not only to arrays, the function checks that the variable is set. So, not sure we should use is_array here, which may be more expensive and redundant. 
As for the strict comparison, absolutely makes sense, thank you.